### Importing librarys

In [24]:
import pandas as pd

### Getting Dataset

In [25]:
df = pd.read_excel('../data/metadados.xlsx')

### Rate analysis

In [26]:
df_rate = df[['id', 'Num Rating']].groupby('Num Rating').count().reset_index()
df_rate.rename(columns={'id': 'quantidade'}, inplace=True)
df['Num Rating'].mean()

4.996666666666667

In [27]:
df_rate

Num Rating  quantidade
0           4           1
1           5         299

### Conclusões

### Base de dados com 300 comentários:
  * 5 estrelas - 299
  * 4 estrelas - 1

### Reviews do produto com 50.622 comentários:
  * 5 estrelas - 43.534
  * 4 estrelas - 4.555
  * 3 estrelas - 1.020
  * 2 estrelas - 505
  * 1 estrela  - 782

### Motivos para a diferença:
  1. Pegando comentários mais recentes, obtém-se dados que:

  * Analisam momento em que o produto já está consolidade no mercado e passou por melhorias
  * Preço baixo durante a black friday
  * Maior qualidade de entrega devido a demanda nesse período
    
  2. Comentários mais negativos apareceriam com um histórico maior de comentários 